www/js/search.js

```diff
@@ -708,6 +732,10 @@ module.exports = {
     banidb.loadBani(BaniID, baniLengthCols[baniLength]).then(rowsDb => {
       // create a unique shabadID for whole bani, and append it with length
       const shabadID = `${rowsDb[0].Token || rowsDb[0].Bani.Token}-${baniLength}`;
+      const nameOfBani = rowsDb[0].Gurmukhi || rowsDb[0].Bani.Gurmukhi;
```

Comment:
Just wondering should the history for Anand sahib (Bhog) say: 
![image](https://user-images.githubusercontent.com/2982784/59321800-42764980-8c88-11e9-930f-0ac40417e396.png)
maybe we need to update db? to keep it consistent?

Reply:
@maneetpaul @saintsoldierx Please advice. _not a blocker_